packages/core/contracts/insured-bridge/BridgePool.sol

```diff
@@ -144,8 +144,7 @@ contract BridgePool is Testable, BridgePoolInterface, ExpandedERC20, MultiCaller
     function addLiquidity(uint256 l1TokenAmount) public {
         l1Token.safeTransferFrom(msg.sender, address(this), l1TokenAmount);
 
-        uint256 lpTokensToMint =
-            FixedPoint.Unsigned(l1TokenAmount).div(FixedPoint.Unsigned(exchangeRateCurrent())).rawValue;
+        uint256 lpTokensToMint = (l1TokenAmount * 1e18) / exchangeRateCurrent();
```

Comment:
This is all possible because solidity 8 uses safe math by default right?

Reply:
yup :)